modules/seqexec/server/src/main/scala/seqexec/server/ObserveActions.scala

```diff
@@ -127,8 +127,7 @@ trait ObserveActions {
       _ <- notifyObserveStart(env).timeoutTo(env.inst.observeTimeout, Concurrent[F].raiseError(SeqexecFailure.ObsSystemTimeout(fileId)))
       _ <- env.headers(env.ctx).traverse(_.sendBefore(env.obsId, fileId))
       _ <- info(s"Start ${env.inst.resource.show} observation ${env.obsId.format} with label $fileId")
-      t <- env.inst.calcObserveTime(env.config).map(t => t + env.inst.observeTimeout)
-      r <- env.inst.observe(env.config)(fileId).timeoutTo(new FiniteDuration(t.millis, MILLISECONDS), Concurrent[F].raiseError(SeqexecFailure.ObsTimeout(fileId)))
+      r <- env.inst.observe(env.config)(fileId)
```

Comment:
Should we  remove the timeouts in the other actions, like pause/abort/etc?

Reply:
Yes, you are right.